lib/Activity/Listener.php

```diff
@@ -70,8 +75,7 @@ protected function setActive(ParticipantModifiedEvent $event): void {
 		$this->roomService->setActiveSince(
 			$event->getRoom(),
 			$this->timeFactory->getDateTime(),
-			$participant->getSession() ? $participant->getSession()->getInCall() : Participant::FLAG_DISCONNECTED,
-			$participant->getAttendee()->getActorType() !== Attendee::ACTOR_USERS
+			$participant->getSession() ? $participant->getSession()->getInCall() : Participant::FLAG_DISCONNECTED
```

Comment:
```suggestion
			$participant->getSession() ? $participant->getSession()->getInCall() : Participant::FLAG_DISCONNECTED,
			$participant->getAttendee()->getActorType() === Attendee::ACTOR_GUESTS
```

To keep writing the data for now, while we fix reading it

Reply:
[Guests are counted from the attendees instead of from the `active_guests` column](https://github.com/nextcloud/spreed/commit/9e55a41e0880219eb993c5690198f159120ccdd5) for several years already, according to the PR:
`Make the call summary reuse the attendee table also for guests, so we don't have "and 132 guests" when a guest rejoins with a weak internet connection`.

Due to that and to either avoid propagating it for consistency, or not propagating it which would work for now if the field is used only for the call summary but could cause unexpected inconsistencies in the future if it ends used for something else, I kept it removed and did not change it.

And if it needs to be introduced back later... we can always do it anyway :-)